src/foam/u2/crunch/UCJReferenceView.js

```diff
@@ -70,6 +71,9 @@ foam.CLASS({
         .remove('RequirementsPreviewAgent')
         .remove('SkipGrantedAgent')
         .remove('WizardStateAgent')
+        .addAfter('SaveAllAgent', this.ApprovalRequestAgent, {
+          group: 'treviso-fraud-ops'
```

Comment:
`treviso-fraud-ops ` isn't generic enough. Perhaps make it default to `fraud-ops` but we'll need to figure out a way to make it configurable to pass in `treviso-fraud-ops` somewhere.

Reply:
This could come from context (a property of the spid capability?) or be a property on the capability itself. I'm leaning toward the first option.